modules/apps/frontend-js/frontend-js-jquery-web/src/main/java/com/liferay/frontend/js/jquery/web/internal/servlet/taglib/JQueryTopHeadDynamicInclude.java

```diff
@@ -74,41 +75,33 @@ public void include(
 		if (themeDisplay.isThemeJsFastLoad()) {
 			sb.append("<script data-senna-track=\"permanent\" src=\"");
 
-			String comboPath = _portal.getStaticResourceURL(
-				httpServletRequest, "/combo", "minifierType=js", _lastModified);
+			ComboRequestAbsolutePortalURLBuilder
+				comboRequestAbsolutePortalURLBuilder =
+					absolutePortalURLBuilder.forComboRequest();
 
-			boolean cdnDynamicResourcesEnabled =
-				_portal.isCDNDynamicResourcesEnabled(
-					themeDisplay.getCompanyId());
```

Comment:
How come we no longer need to check if the CDN is enabled?

Reply:
Because that code is put inside the ComboRequestAbsolutePortalURLBuilder now. 

The idea behind this PR is to make this kind of things unnecessary (in user's side code). All this logic is hidden inside portal-url-builder so that only the infra has to take care of it.

And, for exotic use cases, we provide methods like `ignoreCDNHost()`, `cache()`, etc. so that user's code can tweak default decisions.